README.md

```diff
@@ -2,36 +2,26 @@
 We at [@SchweizerischeBundesbahnen](https://github.com/SchweizerischeBundesbahnen) need to host all the OpenShift events outside our OSE-cluster as it would flood our etcd datastore if we keep the events of all projects for more than one day.
 So this tools just attaches to the kubernetes API and logs all the events to console where they are grabbed and sent to our central logging environment.
 
+# Syslog forwarding
+If SYSLOG_SERVER is defined, we will forward the logs the the syslog server instead of sending the events to the console
 
 # Installation
 ```bash
 # Create a project & a service-account
-oc new-project sbb-infra
+oc project logging
 oc create serviceaccount ose-eventforwarder
 
 # Add a new role to your cluster-policy:
-oc edit clusterPolicy default
-
-###
-- name: ose:eventforwarder
-  role:
-    metadata:
-      creationTimestamp: null
-      name: ose:eventforwarder
-    rules:
-    - apiGroups:
-      - ""
-      attributeRestrictions: null
-      resources:
-      - events
-      verbs:
-      - get
-      - list
-      - watch
-###
+oc create -f deploy/clusterPolicy-forward.yaml
 
 # Add the role to the service-account
-oc adm policy add-cluster-role-to-user ose:eventforwarder system:serviceaccount:sbb-infra:ose-eventforwarder
+oc adm policy add-cluster-role-to-user ose:eventforwarder system:serviceaccount:logging:ose-eventforwarder
+
+# Deploy the new pod
+oc create configmap forward-config \
```

Comment:
Creating a config map and mapping it back to env variable seems a little bit complicated. Why not put the values directly to the env variables?

Reply:
We have many separate OpenShift clusters that we manage.  By using a config map, we only change the config map entries, (as part of our deployment process) and leave the deployment config file the same in all environments.